Rakefile

```diff
@@ -34,7 +34,7 @@ task :delete_sample_solr_docs do
   ss = SparqlToSwSolr::SolrService.new
   require_relative 'config/sample_instance_uris'
   SAMPLE_INSTANCE_URIS.each do |uri|
-    ss.delete_by_id(SparqlToSwSolr::InstanceSolrDoc.instance_uri_to_ckey(uri))
+    ss.delete_by_id(SparqlToSwSolr.abort.instance_uri_to_ckey(uri))
```

Comment:
Not sure how that came about. It complained about `::InstanceSolrDoc` but it appears to work ok now when I changed it back.